google-analytics-setup.html

```diff
@@ -0,0 +1,91 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../google-signin/google-signin-aware.html">
+<link rel="import" href="../google-apis/google-client-loader.html">
```

Comment:
This is a preliminary port, so things are still a bit rough. Looks like 0.8-preview port of client-apis has not been merged yet. The client-apis are at https://github.com/atotic/google-apis/tree/0.8-preview

The reason for initial rough port is that 0.8 is such a moving target, I wanted to get all demos working, and then we can polish it when Polymer gets stable.

I can create a simple <google-analytics-dashboard> that's just an enclosure. It is really not necessary with new code structure, but we can keep it for compatibilty reasons. I removed it because the way it was updating startDate was running into sync problems with 0.8 architecture (updating before elements were upgraded....), and there was a simpler way to do the same thing (properties).

I should be done with all 0.8 elements tonight, then I'll focus on polish.